### PR TITLE
Modifications and Updates to Windows build scripts (+Ledger)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contrib/build-wine/build-electrum-git.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ git pull
 git checkout master
 cd ..
 else
-URL=https://github.com/z-classic/$repo.git
+URL=https://github.com/theapplicationist/$repo.git
```
Comment (nimbosa) [inline contrib/build-wine/build-electrum-git.sh]: please revert this change to your own repo, it should build from the reference repo
`URL=https://github.com/z-classic/$repo.git`

**Review by nimbosa — CHANGES_REQUESTED** [review]
just point to the correct repo, otherwise, this looks good as i see it 👍 @thedevworks Thanks!

**Comment by ghost** [comment]
Done. This just gets a reasonably stable Electrum with ledger build. The right thing to do is rebase on latest Electrum of course, the btchip handling is much better in that. But this is a temporary solution, that worked after much messing around!
